diff --git a/src/lib/wallet/index.ts b/src/lib/wallet/index.ts
index ac1c2a1..1a823fa 100644
--- a/src/lib/wallet/index.ts
+++ b/src/lib/wallet/index.ts
@@ -54,6 +54,9 @@ export {
 	getEncryptedKey,
 	setEncryptedKey,
 	clearEncryptedKey,
+	getEncryptedMnemonic,
+	setEncryptedMnemonic,
+	clearEncryptedMnemonic,
 	getPinHash as getWalletPinHash,
 	setWalletState,
 	getWalletState as getStoredWalletState,
